geo/src/algorithm/bool_ops/tests.rs

```diff
@@ -116,6 +116,39 @@ fn test_complex_rects() -> Result<()> {
     }
     Ok(())
 }
+
+#[test]
+fn single_and_multi() {
+    let wkt1 = "POLYGON ((110 310, 220 310, 220 210, 110 210, 110 310))";
+    // multipolygon containing a single polygon
+    let wkt2 = "MULTIPOLYGON (((80 260, 90 260, 90 250, 80 250, 80 260)))";
+    // From JTS union op
+    let res = "MULTIPOLYGON (((110 310, 220 310, 220 210, 110 210, 110 310)), ((80 260, 90 260, 90 250, 80 250, 80 260)))";
+    let poly = Polygon::<f64>::try_from_wkt_str(wkt1).unwrap();
```

Comment:
```suggestion
    let poly = geo::wkt!(POLYGON ((110 310, 220 310, 220 210, 110 210, 110 310)));
```

reminder about the geo::wkt! macro: https://docs.rs/geo/latest/geo/macro.wkt.html

Reply:
lol I'm sorry that I sabotaged you. 

The macro will try to build an integer geometry if you give it integers. Forget it!

Reply:
Oh noooooo